### DISCUSSION TITLE
[Feature] Add Pictures to multiple albums in one go

### DISCUSSION THREAD

**Discussion by MaroIshiku** [immich-app/immich#12029] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

If you want to add a picture/video to multiple albums, you have to add them to one album at a time.
I believe it would improve user experience, if you would be able to mark several albums by long pressing one and confirm to add the pictures to them.
I tried creating an example, but I am no photoshop pro:

Web:
![Screenshot_20240825-152126](https://github.com/user-attachments/assets/6beb404b-0e78-4a2b-bca5-b665fc8d4ddb)

Mobile:
![Screenshot_20240825-151624](https://github.com/user-attachments/assets/3feec7e5-5551-4bec-817f-ef8f019a1310)

### Platform

- [ ] Server
- [x] Web
- [x] Mobile

**Comment by bo0tzz** [comment]
#2686

**Comment by wtangofoxtrot** [comment]
https://github.com/immich-app/immich/discussions/2686 is not what the request here appears to be @bo0tzz, its asking for the ability to upload to multiple albums in one go. This has nothing to do with https://github.com/immich-app/immich/discussions/2686 which is about retaining the selected photos. 

As an example in Nextcloud when uploading photos you can select multiple albums to share the photo amongst in one go and even create new albums at before uploading in the same dialog. 
![Screenshot 2025-02-20 at 9 58 06 PM](https://github.com/user-attachments/assets/85ba472d-d951-4aaa-84d4-f8944117c363)
![Screenshot 2025-02-20 at 9 58 19 PM](https://github.com/user-attachments/assets/c5509dae-1d5c-46f6-93ef-1b70f72c33c9)

**Comment by Gabriele-tomai00** [comment]
It should be noted that a image can be in multiple albums at the same time. If the image is already in an album, you can still use the “add to album” function.
This leaves me a little perplexed, as I am used to using folders, and so I don't understand whether a “move to album” function makes sense (but I would like it!)

I just realised this: I have a photo in the WhatsApp album (it backs up), I can take the image and add it to another album, and then it appears in two albums, but if I delete it from one album, it is permanently deleted! I don't understand the logic, the photo should be either in one album or another, and if I delete it, it should be deleted.

**Comment by bo0tzz** [comment]
Coming back to this again, this was reopened incorrectly and is indeed a dupe of #2686
